backend/src/common/services/azure-ai/apis/text-to-speech-api.ts

```diff
@@ -0,0 +1,37 @@
+import { config } from '~/common/config/config.js';
+import { BaseHttpApi } from '~/common/http/base-http-api.js';
+import { type Http, ContentType, HTTPMethod } from '~/common/http/http.js';
+
+import { TextToSpeechApiPath } from '../enums/enums.js';
+import { type AzureGetVoicesResponseDto } from '../types/types.js';
+
+type Constructor = {
+    baseUrl: string;
+    http: Http;
+};
+
+class TextToSpeechApi extends BaseHttpApi {
+    public constructor({ baseUrl, http }: Constructor) {
+        super({ path: '', baseUrl, http });
+    }
+
+    public async getVoices(): Promise<Array<AzureGetVoicesResponseDto>> {
+        const response = await this.load(
+            this.getFullEndpoint(TextToSpeechApiPath.VOICES, {}),
+            {
+                method: HTTPMethod.GET,
+                contentType: ContentType.JSON,
+                headers: [
+                    {
+                        key: 'Ocp-Apim-Subscription-Key',
```

Comment:
Use constants here, and check my PR for reference.
https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/199

Reply:
Done